_data-prepper/managing-data-prepper/source-coordination.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: default
+title: Source coordination
+nav_order: 35
+parent: Managing Data Prepper
+---
+
+# Source coordination
+
+_Source coordination_ is the concept of coordinating and distributing work between Data Prepper data sources in a multi-node environment. Some data sources, such as Amazon Kinesis or Amazon Simple Queue Service (Amazon SQS), handle coordination natively. Other data sources, such as OpenSearch, Amazon Simple Storage Service (Amazon S3), Amazon DynamoDB, and JDBC/ODBC, do not support source coordination.
+
+Data Prepper source coordination decides which partition of work is performed by each node in the Data Prepper cluster and prevents duplicate partitions of work once the node completes work.
```

Comment:
How about this? @natebower 

```suggestion
Data Prepper source coordination decides which partition of work is performed by each node in the Data Prepper cluster and prevents duplicate partitions of work after.
```

Reply:
Per our discussion, if the sentence can end at "work" and not lose its meaning, I'm fine with that.